public/app/features/dashboard/components/VersionHistory/HistoryListCtrl.ts

```diff
@@ -2,28 +2,22 @@ import _ from 'lodash';
 import angular, { ILocationService, IScope } from 'angular';
 
 import { DashboardModel } from '../../state/DashboardModel';
-import { CalculateDiffOptions, HistoryListOpts, HistorySrv, RevisionsModel } from './HistorySrv';
-import { AppEvents, DateTimeInput, locationUtil } from '@grafana/data';
+import { DecoratedRevisionModel } from '../DashboardSettings/VersionsSettings';
+import { CalculateDiffOptions, HistorySrv } from './HistorySrv';
+import { AppEvents, locationUtil } from '@grafana/data';
 import { GrafanaRootScope } from 'app/routes/GrafanaCtrl';
 import { CoreEvents } from 'app/types';
 import { promiseToDigest } from '../../../../core/utils/promiseToDigest';
-import { appEvents } from 'app/core/app_events';
 
 export class HistoryListCtrl {
```

Comment:
I guess this one is left until we migrate the diff to react right?

Reply:
I don't believe it's required as linting gave me `declared but never read` warnings so I deleted it. ✂️